src/main/java/jeriel/util/Storage.java

```diff
@@ -0,0 +1,84 @@
+package jeriel.util;
+import jeriel.command.*;
+import jeriel.task.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        // System.out.println("Saving to: " + new File(filePath).getAbsolutePath());
+
+    }
+
+    
+
+    /**
+     * Loads the tasks from the file specified by the file path.
+     *
+     * The file is expected to have the following format:
+     * T | 0/1 | description
+     * D | 0/1 | description | by
+     * E | 0/1 | description | from | to
+     *
+     * @return a list of tasks
+     * @throws IOException if there is an error reading from the file
+     */
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        File file = new File(filePath);
+
+        if (!file.exists()) {
+            file.getParentFile().mkdirs();
+            file.createNewFile();
+        }
+
+        Scanner scanner = new Scanner(file);
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            System.out.println("Reading line: " + line); // Debugging line
+            String[] parts = line.split(" \\| ");
+            System.out.println("Split parts: " + Arrays.toString(parts)); // Debugging line
+
+            Task task;
+
+            switch (parts[0]) {
+                case "T":
```

Comment:
If the save file is corrupted, will this still work? (Like the save file has just a "T")